scripts/dataset_processing/tts/sfbilingual/ds_conf/ds_for_fastpitch_align.yaml

```diff
@@ -0,0 +1,47 @@
+name: "ds_for_fastpitch_align"
+
+manifest_filepath: "train_manifest.json"
+sup_data_path: "sup_data"
+sup_data_types: [ "align_prior_matrix", "pitch" ]
+whitelist_path: "nemo_text_processing/text_normalization/zh/data/whitelist/default.tsv"
+phoneme_dict_path: "scripts/tts_dataset_files/zh/pinyin_dict.txt"
+
+dataset:
+  _target_: nemo.collections.tts.torch.data.TTSDataset
+  manifest_filepath: ${manifest_filepath}
+  sample_rate: 22050
+  sup_data_path: ${sup_data_path}
+  sup_data_types: ${sup_data_types}
+  n_fft: 1024
+  win_length: 1024
+  hop_length: 256
+  window: "hann"
+  n_mels: 80
+  lowfreq: 0
+  highfreq: null
+  max_duration: null
+  min_duration: 0.1
+  ignore_file: null
+  trim: false
```

Comment:
I would suggest applying the same trim config as `fastpitch_align_22050.yaml` above, otherwise, it may diverge the pitch norm slightly.